apps/web/src/app/[locale]/tracks/_components/track-card.tsx

```diff
@@ -19,21 +20,39 @@ export const BGS_BY_TRACK: Record<number, string> = {
 } as const;
 export const bgsArray = Object.values(BGS_BY_TRACK);
 
-const EnrolledBadge = ({ text = 'Enrolled' }: { text?: string }) => (
-  <div className="text-background absolute left-0 top-0 z-10 rounded-br-3xl bg-[#3078c5] px-5 py-1.5 text-sm font-bold dark:bg-blue-400">
+const EnrolledAndCompletedBadge = ({ text = 'Enrolled' }: { text?: string }) => (
+  <div
+    className={clsx(
+      'text-background absolute left-0 top-0 z-10 rounded-br-3xl bg-[#3078c5] px-5 py-1.5 text-sm font-bold dark:bg-blue-400',
+      text === 'Completed' && 'bg-green-500 text-white dark:bg-green-700',
```

Comment:
Use the darker color ‘bg-green-700’ for light mode and the lighter color ‘bg-green-500’ for dark mode.